tracekit.js

```diff
@@ -248,6 +262,42 @@ TraceKit.report = (function reportModuleWrapper() {
         _onErrorHandlerInstalled = true;
     }
 
+    /**
+     * Uninstall the global onerror handler
+     * @memberof TraceKit.report
+     */
+    function uninstallGlobalHandler() {
+        if (_onErrorHandlerInstalled) {
+            window.onerror = _oldOnerrorHandler;
+            _onErrorHandlerInstalled = false;
+        }
+    }
+
+    /**
+     * Install a global onunhandledrejection handler
+     * @memberof TraceKit.report
+     */
+    function installGlobalUnhandledRejectionHandler() {
+        if (_onUnhandledRejectionHandlerInstalled === true) {
+            return;
+        }
+
+        _oldOnunhandledrejectionHandler = window.onunhandledrejection;
+        window.onunhandledrejection = traceKitWindowOnUnhandledRejection;
+        _onUnhandledRejectionHandlerInstalled = true;
+    }
+
+    /**
+     * Uninstall the global onunhandledrejection handler
+     * @memberof TraceKit.report
+     */
+    function uninstallGlobalUnhandledRejectionHandler() {
+        if (_onUnhandledRejectionHandlerInstalled) {
+            window.onerror = _oldOnunhandledrejectionHandler;
```

Comment:
@niemyjski @ghostd shouldn't this be

```js
window.onunhandledrejection = _oldOnunhandledrejectionHandler;
```

Right now it looks like it's a copy-paste error.

If it indeed should be `window.onerror = _oldOnunhandledrejectionHandler;` can you please explain why?

Reply:
If it really is a typo then there is https://github.com/csnover/TraceKit/pull/74 to fix it

Reply:
You're right, the "uninstallation" function is bugged. Thx for the PR